app/views/index.1.html

```diff
@@ -0,0 +1,176 @@
+{% set title = 'Home' %}
```

Comment:
Moved the original index so it can be referred back during the build. Good!

Reply:
I'd probably just remove it? There are other ways we can revisit it.

Reply:
We can still have them in the repo for easy access, though.